switchbot_ros/scripts/switchbot_ros_server.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import rospy
+import actionlib
+from switchbot_ros.msg import SwitchBotCommandAction, SwitchBotCommandFeedback, SwitchBotCommandResult
+from switchbot import SwitchBotAPIClient
+
+
+class SwitchBotAction:
+    """
+    Control your switchbot with ROS and SwitchBot API
+    """
+    _feedback = SwitchBotCommandFeedback()
+    _result = SwitchBotCommandResult()
```

Comment:
It is better to make a message instance in every loop to avoid bug from non-initialization.